src/Model/Search.php

```diff
@@ -29,4 +39,207 @@ public static function getUserTags()
 
 		return $tags;
 	}
+
+	/**
+	 * Search a user based on his/her profile address
+	 * pattern: @username@domain.tld
+	 *
+	 * @param string $user The user to search for
+	 *
+	 * @return ResultList|null
+	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
+	 * @throws \ImagickException
+	 */
+	public static function searchUser($user)
```

Comment:
I'm not a fan of this method being there for two reasons:
- it doesn't have anything to do with the `search` table
- The full name is `Search::searchUser` which is repetitive and an indication this method might be misplaced.

Since it seems to be related with contacts, I think it should be moved to `Model\Contact`.